src/storage/metrics.rs

```diff
@@ -48,12 +92,15 @@ lazy_static! {
     .unwrap();
     pub static ref KV_COMMAND_COUNTER_VEC_STATIC: KvCommandCounterVec =
         KvCommandCounterVec::from(&KV_COMMAND_COUNTER_VEC);
-    pub static ref SCHED_STAGE_COUNTER_VEC: IntCounterVec = register_int_counter_vec!(
-        "tikv_scheduler_stage_total",
-        "Total number of commands on each stage.",
-        &["type", "stage"]
-    )
-    .unwrap();
+    pub static ref SCHED_STAGE_COUNTER_VEC: SchedStageCounterVec = {
```

Comment:
why the '{}' is needed here?

Reply:
Just for format reason, split the long line.